simpeg/optimization.py

```diff
@@ -436,7 +441,11 @@ def startup(self, x0):
         self.iterLS = 0
         self.stopNextIteration = False
 
-        x0 = self.projection(x0)  # ensure that we start of feasible.
+        try:
+            x0 = self.projection(x0)  # ensure that we start of feasible.
+        except Exception as err:
+            raise RuntimeError("Initial model is not projectable") from err
+
```

Comment:
This is to add a slightly improved message for `Bounded` problems, will be thrown if initial model does not broadcast with the upper or lower bounds.